src/Apps/Artwork/Components/ArtworkSidebar/__stories__/ArtworkSidebarBidActionStory.story.tsx

```diff
@@ -25,135 +28,415 @@ storiesOf("Apps/Artwork/Components/Sidebar", module).add("BidAction", () => {
       </Section>
-      <Section title="Live auction / Open registration / Bidder pending approval">
+
+      <Section title="Live auction / Open Registration / Bidder Pending Approval">
         <BidAction
           artwork={merge(
             {},
             ArtworkFromLiveAuctionRegistrationOpen,
             BidderPendingApproval
           )}
+          me={NotIDVedUser as any}
         />
       </Section>
-      <Section title="Live auction / Closed registration / Bidder pending approval">
+
+      <Section title="Live auction / Requires Identity Verification / Open Registration / Bidder Pending Approval / Not Identity Verified">
         <BidAction
           artwork={merge(
             {},
-            ArtworkFromLiveAuctionRegistrationClosed,
+            ArtworkFromLiveAuctionRegistrationOpen,
+            { sale: SaleRequiringIDV },
+            BidderPendingApproval
+          )}
+          me={NotIDVedUser as any}
+        />
+      </Section>
+
+      <Section title="Live auction / Requires Identity Verification / Open Registration / Bidder Pending Approval / Identity Verified">
+        <BidAction
+          artwork={merge(
+            {},
+            ArtworkFromLiveAuctionRegistrationOpen,
+            { sale: SaleRequiringIDV },
             BidderPendingApproval
           )}
+          me={IDVedUser as any}
         />
       </Section>
-      <Section title="Live auction / Open registration / Registered Bidder">
+
+      <Section title="Live auction / Open Registration / Registered Bidder">
         <BidAction
           artwork={merge(
             {},
             ArtworkFromLiveAuctionRegistrationOpen,
             RegisteredBidder
           )}
+          me={NotIDVedUser as any}
         />
       </Section>
-      <Section title="Live auction / Closed registration / Registered Bidder">
+
+      <Section title="Live auction / Closed registration / No Bidder">
+        <BidAction
+          artwork={merge(
+            {},
+            ArtworkFromLiveAuctionRegistrationClosed,
+            NotRegisteredToBid
+          )}
+          me={NotIDVedUser as any}
+        />
+      </Section>
+
+      <Section title="Live auction / Closed registration / Bidder Pending Approval">
+        <BidAction
+          artwork={merge(
+            {},
+            ArtworkFromLiveAuctionRegistrationClosed,
+            BidderPendingApproval
+          )}
+          me={NotIDVedUser as any}
+        />
+      </Section>
+
+      <Section title="Live auction / Closed Registration / Registered Bidder">
         <BidAction
           artwork={merge(
             {},
             ArtworkFromLiveAuctionRegistrationClosed,
             RegisteredBidder
           )}
+          me={NotIDVedUser as any}
         />
       </Section>
 
       {/* Auction in Closed */}
       <Section title="Auction closed">
         <BidAction
           artwork={merge({}, ArtworkFromClosedAuction, RegisteredBidder)}
+          me={NotIDVedUser as any}
```

Comment:
🙏 thanks for all of these storybooks configurations. It will be good to evangelize these just in case someone on the team only works from a linked Force. Might save a lot of trouble 😄